### ISSUE TITLE
pi2040-zero can't find micro sd

### ISSUE THREAD

**Issue by MrPalloncini** [dangiu/PicoMemcard#37]
Hi, i soldered and troubleshooted everyting  by following the readme, howewer, when i connect the rp2040-0 the the pc after i flasched it, the sd won't appear and it flash red. Tryied also with a raspberry pi pico and it works, is there any problem with the fw maybe?

Thank you for thia awesome project

**Comment by dangiu** [comment]
No, the firmware has been tested by many people, normally when the SD card does not appear is related either to a power problem with the module or due to the card not being formatted in FAT32. Since you said that it works fine on the pico I think it may be due to a power problem, make sure to double check your wiring and the voltage the module requires... It could be that the pico and rp2040-zero are outputting slightly different voltages.
Also check out issue #36 which discusses power related issue with modules.

**Comment by seanchapman** [comment]
I'm having this issue too, as well as the inverted red and green LEDs (on a recent RP2040). Firmware v1.0.0 works but v1.0.2 does not.

On v1.0.2 I see the removable drive in Windows but it is empty i.e. "no disk". If I put it in the PSX it flashes green (which I think is meant to be red, i.e. can't read SD card).

On v1.0.0 the removable drive shows in Windows and I can see the memory card files. On the PSX it goes red (which is meant to be green) and the files load. The LED occassionally turns off though or doesnt turn on.

**Comment by dangiu** [comment]
About the LED problem, have you looked into issue #38? It seems like you are not the only one having it.
Regarding the disk instead I have no clue what may be going on? Can you provide some more info (what works on which platform (OS) and what does not with the different firmware versions)? By chance are you able to debug the program and step through the code?

**Comment by seanchapman** [comment]
I've just tested the inverted LED build and actually that's working great for me now! LED is the correct colour, saves are loading on PSX and I can see the microSD in Windows.

I tried v1.0.2 again as well and that seems to work now too so I'm not sure what happened with that.
Previously I was using v1.0.0 and having the issues with the LED turning off.

I'm not familiar with pico debugging but it seems like the inverted build is working best for me.
